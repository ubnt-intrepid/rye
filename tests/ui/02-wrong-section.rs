fn main() {}

#[rye::test_case]
fn one_section() {
    section!();
}

#[rye::test_case]
fn multi_sections() {
    section!();
    section!();
}

#[rye::test_case]
fn nested_sections() {
    section!("a", {
        section!();
    });
}
