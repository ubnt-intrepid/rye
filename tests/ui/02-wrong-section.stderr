error: unexpected end of input, expected expression
 --> $DIR/02-wrong-section.rs:5:13
  |
5 |     section!();
  |             ^^

error: unexpected end of input, expected expression
  --> $DIR/02-wrong-section.rs:10:13
   |
10 |     section!();
   |             ^^

error: unexpected end of input, expected expression
  --> $DIR/02-wrong-section.rs:11:13
   |
11 |     section!();
   |             ^^

error: unexpected end of input, expected expression
  --> $DIR/02-wrong-section.rs:17:17
   |
17 |         section!();
   |                 ^^
